lib/std/Build/Step/Compile.zig

```diff
@@ -1865,7 +1858,11 @@ fn addFlag(args: *ArrayList([]const u8), comptime name: []const u8, opt: ?bool)
     }
 }
 
+<<<<<<< HEAD
```

Comment:
It looks like something went wrong here.

Reply:
Fixed, missed that during the rebase.